integration/map.go

```diff
@@ -85,7 +86,7 @@ func RunMapIntegration(ctx context.Context, mapID int64, client trillian.Trillia
 
 	// Check values
 	// Mix up the ordering of requests
-	h := merkle.NewMapHasher(testonly.Hasher)
+	h := rfc6962.DefaultHasher
```

Comment:
In that case possibly more stuff should move out of rfc6962. We don't want to move generic code into it.